### DISCUSSION TITLE
Linux6.2内核的配置文件可以分享一下吗？

### DISCUSSION THREAD

**Discussion by 2004none** [vrolife/modern_laptop#80] [Q&A]
我发现我的apt不能搜到6.2.0以上的内核，只能自己编译了，然后发现8852BE是默认不编译的，不知道还有哪些模块没有选。。。。求大佬分享

**Comment by vrolife** [comment]
去/boot下找6.1的配置来用，https://kernel.ubuntu.com/~kernel-ppa/mainline/ 这里也有现成的。
